### PR TITLE
feat: delete user documentation

### PR DESCRIPTION
**Pull request by Inksprout**
The purpose of this PR is to document the delete user feature that is being added to support SCIM integration. The delete functionality will behave similarly to the disable user functionality but once delete the user will not show up in the UI anymore.

### CODE REVIEW

File: website/docs/docs/user-interface/settings/users.md [hunk 1]
```diff
@@ -38,6 +39,14 @@ Bulk actions can be applied to multiple users:
 
 _NOTE: disabled users do not count toward your user limit._
 
+## Delete User
+
+You may delete a user by a user by either choosing the action via the `"..."` menu to the right of the user's record or by selecting the user with the checkbox and using the bulk action menu. These actions apply to local users or federated identities (such as SAML2.0, Google or GitHub providers).
```
Comment (bethesque) [inline website/docs/docs/user-interface/settings/users.md:44]: Typo - extra `a user by`

**Comment by bethesque** [comment]
Should we say that if a user is desired to remain in the org's SSO, but should not have pactflow access, they should disable the user, rather than deleting it?

**Comment by Inksprout** [comment]
> Should we say that if a user is desired to remain in the org's SSO, but should not have pactflow access, they should disable the user, rather than deleting it?

Yeah I am thinking that. Would be interesting to see if we get some feedback around this but I think that makes the most sense to me currently as a recommendation

**Comment by bethesque** [comment]
I can imagine that a company might want to do this because they wanted to reduce the number of users they were paying for in the license. Deactivating but not deleting a user would be one way to achieve that. But yes, definitely worth getting feedback on.

**Comment by bethesque** [comment]
Those images are nice!

**Pull request merged**